applications/nrf_desktop/src/modules/ble_scan.c

```diff
@@ -407,6 +414,26 @@ static void scan_init(void)
 	k_delayed_work_init(&scan_stop_trigger, scan_stop_trigger_fn);
 }
 
+static void enable_llpm(struct bt_conn *conn)
+{
+	if (IS_ENABLED(CONFIG_BT_LL_NRFXLIB)) {
+		struct bt_le_conn_param param = {
+			.interval_min = 0x0D01,
```

Comment:
When connecting two devices, we should use a connection interval of 2 ms, as each connection reserves 1 ms

Reply:
Let's leave it at 1 since it should work correctly and we don't need to update params in case one of the devices leaves.
We will get back to it while doing power measurements. I don't expect it will be important for out mouse as most power goes to LEDs but it may be something worth checking for keyboard. For dongle we don't care as it is powered from USB.